modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManager.vue

```diff
@@ -230,12 +239,6 @@ export default {
     this.modal.active = true;
     this.getPieces();
   },
-  watch: {
-    // NOTE: revisit this during refactoring
-    checked: function() {
```

Comment:
this is needed for the relationship editor

Reply:
It's still there. The linter wants these in a particular order, so it's moved up to line 226.

Reply:
Ok